### PR TITLE
Test against newer python versions

### PR DESCRIPTION
**Pull request by gforcada**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 language: python
 python:
   - '2.7'
```
Comment (jkowalleck) [inline .travis.yml:3]: end of life

File: .travis.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
-  - '3.5'
```
Comment (jkowalleck) [inline .travis.yml:4]: still active

File: .travis.yml [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
   - '3.6'
+  - '3.7'
```
Comment (jkowalleck) [inline .travis.yml:5]: nice. could you add py '3.8' ?

File: .travis.yml (hunk 1 above)
Comment (gforcada) [reply .travis.yml:3]: Yes, but still being widely used, it does not harm anyone to keep testing it. I might remove testing later next year. Thanks for the reminder though!

File: .travis.yml (hunk 3 above)
Comment (gforcada) [reply .travis.yml:5]: We can not as long as `cffi`, one development dependency we have supports it as well.

File: .travis.yml (hunk 2 above)
Comment (gforcada) [reply .travis.yml:4]: right, but I don't want to test on every single version just for the sake of it, I'm keeping 2.7 a while longer, and I will keep testing the last 2-3 3.x releases only.

**Pull request merged**